README.md

```diff
@@ -8,22 +8,23 @@ It currently consists of three classes:
 * **WikiFile** – Provides an interface to downloading/uploading files and getting their properties.
 
 The [latest released version](https://github.com/hamstar/Wikimate/releases) of Wikimate
-is v0.15.0, released on Aug 26, 2021. It requires MediaWiki v1.27 or newer.
+is v0.15.0, released on Aug 26, 2021.
```

Comment:
This is what I had in mind, because this is the only clause to change every release.